### ISSUE TITLE
lighthouse组件是必须的吗

### ISSUE THREAD

**Issue by Ethan-rookie** [Tencent/caelus#53]
请教下离线作业在yarn上，not on k8s，是不是可以不用部署lighthouse和plugin server

**Comment by Ethan-rookie** [comment]
@ddongchen

**Comment by ddongchen** [comment]
> 请教下离线作业在yarn上，not on k8s，是不是可以不用部署lighthouse和plugin server

 lighthouse和plugin server是通过拦截方式加强离线容器启动的隔离性，离线作业在yarn上运行，yarn的nm是通过k8s pod部署，也需要这两个组件。

**Comment by Ethan-rookie** [comment]
> > 请教下离线作业在yarn上，not on k8s，是不是可以不用部署lighthouse和plugin server
> 
> lighthouse和plugin server是通过拦截方式加强离线容器启动的隔离性，离线作业在yarn上运行，yarn的nm是通过k8s pod部署，也需要这两个组件。

多谢，还想请教下自定义的服务指标，比如qps，rule check需要怎么配置呢？

**Comment by ddongchen** [comment]
> > > 请教下离线作业在yarn上，not on k8s，是不是可以不用部署lighthouse和plugin server
> > 
> > 
> > lighthouse和plugin server是通过拦截方式加强离线容器启动的隔离性，离线作业在yarn上运行，yarn的nm是通过k8s pod部署，也需要这两个组件。
> 
> 多谢，还想请教下自定义的服务指标，比如qps，rule check需要怎么配置呢？

自定义的服务指标可参考https://github.com/Tencent/caelus/blob/master/doc/config.md#8online
目前是配置在app模块里。这块正在考虑单独拿出来，扩展性更强。

**Comment by Ethan-rookie** [comment]
> 

好的，“在线作业非运行在K8s平台相关配置”这句话看起来是配置的是online_type:local类型的在线作业，online_type:k8s同样也可以配置指标是吧？

**Comment by ddongchen** [comment]
> > 
> 
> 好的，“在线作业非运行在K8s平台相关配置”这句话看起来是配置的是online_type:local类型的在线作业，online_type:k8s同样也可以配置指标是吧？

是的。若在线作业为非k8s，即直接裸机部署进程，则online_type:local。若在线通过k8s提交，则online_type:k8
